site/content/en/docs/drivers/includes/none_usage.inc

```diff
@@ -1,8 +1,9 @@
 ## Requirements
 
-VM running a systemd-based Linux distribution ([see #2704](https://github.com/kubernetes/minikube/issues/2704))
-
-kubeadm requirements listed here - https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
+* VM running a systemd-based Linux distribution ([see #2704](https://github.com/kubernetes/minikube/issues/2704))
+  - OpenRC systems are supported on master (to be released in minikube v1.10)
```

Comment:
Not sure how the suggestion I made is really going to render on the site, but maybe we want OpenRC statement to be a part of the same bullet point, just on a separate line?
```suggestion
* VM running a systemd-based Linux distribution ([see #2704](https://github.com/kubernetes/minikube/issues/2704))  
OpenRC systems are supported on master (to be released in minikube v1.10)
```